apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -294,7 +311,8 @@ model ActivityType {
   users            UserActivity[]
   contributions    Contribution[]
   categoryActivity CategoryActivityType[]
-  guilds           GuildActivityType[]
+  guilds           GuildActivityType[]    @relation(name: "guilds")
```

Comment:
explicit management here is required. It might have changed when we upgraded to prisma 3, but there were implicit relations allowed in prisma 2 that are now disallowed in 3.

specifically, prisma 3 is not allowing two different fk references from GuildActivityType to ActivityType. See the below error:

![image](https://user-images.githubusercontent.com/13227294/234052893-fabac2d2-dd77-47af-9728-8fc3fff9eee2.png)

Reply:
I guess this crucial to let prisma know what's type of this relation and for generated ts files. 

I would like to ask. Do we need to limit the functionality of the opposite direction queries?

Reply:
I don't think so. i think this is a convenience field. but I don't fully understand how this works with prisma, i admit. I propose we take any functionality limiting we need to do as a separate PR.